src/functions/getTestCentreJournal/application/helpers/helpers.ts

```diff
@@ -84,6 +86,14 @@ const inNext2Days = <T>(section: T): boolean => {
   return today || tomorrow;
 };
 
+export const nonADI2Slots = (section: TestSlot): boolean => {
```

Comment:
is this name descriptive of what the func does? seems something like `isNonADI2TestSlot` would be more apt?